pkg/env/costmodelenv.go

```diff
@@ -171,12 +169,6 @@ func GetMetricsConfigmapName() string {
 	return Get(MetricsConfigmapName, "metrics-config")
 }
 
-// GetAWSAccessKeyID returns the environment variable value for AWSAccessKeyIDEnvVar which represents
-// the AWS access key for authentication
-func GetAppVersion() string {
-	return Get(AppVersionEnvVar, "1.91.0-rc.0")
-}
-
```

Comment:
Since we're nuking this (which I think is the right call due to build args existing), would you please open a matching PR in the Helm chart that removes the env var from there?

Reply:
Good. @nealormsbee may want to check that sentry setup anyway then eh? We may have been dropping stuff but shouldn't in the future.